salesagility/SuiteCRM
# Fix issue with saving record when reply to flag is checked

Checking "reply to" on a user email address generates a "Bad data - return to Home" error.


Removed an extra double-quote

## Description




## Motivation and Context
Saving a user profile with Reply-to checked on an email address generates a "Bad data - return to Home" error

## How To Test This
- Install SuiteCRM with demo data
- Edit one of the users profile
- check Reply-to on an email address
- Save the user

## Types of changes


### Final checklist
